### PR TITLE
made year and month dynamic

### PR DESCRIPTION
**Pull request by Abhishek25081999**
Targets #2442

### Description
Made year and Month dynamic

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Revenue/Resources/views/index.blade.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
             @foreach ($revenueProceeds as $revenueProceed)
                 <tr>
                     <td>{{ $revenueProceed->name }}</td>
-                    <td>{{Str::title(str_replace('-', ' ', $revenueProceed->category))}}</td>
+                    <td>{{Str::title(str_replace('_', ' ', $revenueProceed->category))}}</td>
```
Comment (pankaj-ag) [inline Modules/Revenue/Resources/views/index.blade.php:28]: Can we use Str::title or something like that?

**Review by pankaj-ag — CHANGES_REQUESTED** [review]
Also when we create revenue can we set the month, year and receive the date as per the current day?

File: Modules/Revenue/Resources/views/index.blade.php (hunk 1 above)
Comment (Abhishek25081999) [reply Modules/Revenue/Resources/views/index.blade.php:28]: ok I'll try.

Comment (pankaj-ag) [reply Modules/Revenue/Resources/views/index.blade.php:28]: Yes @Abhishek25081999  you can use something from this list. 
https://laravel.com/docs/9.x/helpers#strings-method-list

Comment (Abhishek25081999) [reply Modules/Revenue/Resources/views/index.blade.php:28]: I think we can. I took reference from other files and its being used there already.

Comment (pankaj-ag) [inline Modules/Revenue/Resources/views/index.blade.php:28]:
```suggestion
                    <td>{{ Str::title($revenueProceed->category) }}</td>
```

**Review by pankaj-ag — APPROVED** [review]
Last feedback then we are good to merge.

**Pull request merged**